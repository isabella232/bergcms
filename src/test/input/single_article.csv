1	chapter	-1	title	K	 head	 body	 footer	20111006-212548-Do / 127.0.0.1:63880
