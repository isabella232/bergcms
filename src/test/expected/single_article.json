Content-type: application/json; charset=utf-8

{
    "id": 1,
    "priority": -1,
    "type": "K",
    "chapter": "chapter",
    "title": "title",
    "header": "head",
    "body": "body",
    "footer": "footer",
    "lastChanged": "20111006-212548-Do / 127.0.0.1:63880"
}
